### DISCUSSION TITLE
Link to the actual Plausible dashboard in the "Check your stats" email

### DISCUSSION THREAD

**Discussion by RobertJoonas** [plausible/analytics#1438] [Anything Else]
Currently, the "Check your stats" email does not mention the actual site name and the link "View your Plausible dashboard now" simply links to plausible.io.

Would be helpful to link to the actual Plausible dashboard of this site, so that people on my team would know which site it actually is.

**Comment by jprangenbergde** [comment]
I have the same problem.

https://twitter.com/jprangenbergde/status/1455959203718250501

**Comment by metmarkosaric** [comment]
Same request for "Login to view your dashboard" in monthly/weekly reports. Right now they link to the login rather than to the actual dashboard. 

The problem is that the dashboard link shows 404 error when people are not logged in so that should be solved first (for instance by redirecting to the home page or to the login page for non-logged in users) and then we can link to the dashboard directly in emails too.

https://github.com/plausible/analytics/discussions/1139
